src/components/ChallengeEditor/Forum-Field/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Select from '../../Select'
+import cn from 'classnames'
+import styles from './Forum-Field.module.scss'
+
+const ForumField = ({ types, onSelectForum, challenge, hasForum, disabled }) => {
```

Comment:
I don't see using ForumField anywhere. I think you are using Discussion-Field instead.